From 33fde4a6bf5647967dbcc391bb8666b41b4c85ca Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E5=BF=98=E6=80=80?= <guanwentao@uniontech.com>
Date: Wed, 31 Jul 2024 23:24:12 +0800
Subject: [PATCH 195/360] [Deepin-Kernel-SIG] disable ARM_PSCI_CHECKER &&
 ARM_PSCI_CPUIDLE
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Log:

We should support Phytium FT2000/4、D2000/8、、etc old arm cpu.
For compat old cpu, their PSCI idle are broken.
If we use PSCI_CHECKER to detect, it caused some FT2000/4
hotplug failed and three core not online.
---
 arch/arm64/configs/deepin_arm64_desktop_defconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/configs/deepin_arm64_desktop_defconfig b/arch/arm64/configs/deepin_arm64_desktop_defconfig
index 427be175a870..63912e39f94e 100644
--- a/arch/arm64/configs/deepin_arm64_desktop_defconfig
+++ b/arch/arm64/configs/deepin_arm64_desktop_defconfig
@@ -114,7 +114,6 @@ CONFIG_ENERGY_MODEL=y
 CONFIG_CPU_IDLE_GOV_LADDER=y
 CONFIG_CPU_IDLE_GOV_MENU=y
 CONFIG_CPU_IDLE_GOV_TEO=y
-CONFIG_ARM_PSCI_CPUIDLE=y
 CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_STAT=y
 CONFIG_CPU_FREQ_DEFAULT_GOV_USERSPACE=y
@@ -826,7 +825,6 @@ CONFIG_EFI_ZBOOT=y
 CONFIG_EFI_BOOTLOADER_CONTROL=m
 CONFIG_EFI_CAPSULE_LOADER=m
 CONFIG_RESET_ATTACK_MITIGATION=y
-CONFIG_ARM_PSCI_CHECKER=y
 CONFIG_TEGRA_IVC=y
 CONFIG_GNSS=m
 CONFIG_GNSS_MTK_SERIAL=m
-- 
2.20.1

